content/riak/kv/2.0.7/using/reference/search.md

```diff
@@ -402,3 +403,46 @@ ownership change this could mean a delay between cluster state and the
 cached plan. This is, however, a good trade-off given the performance
 benefits, especially since even without caching there is a race, albeit
 one with a smaller window.
+
+## Statistics
+
+The Cuttlefish batching subsystem maintains a set of statistics that provide visibility into the run-time characteristics of the components that make up the system. These statistics are accessible via the standard Riak KV stats interfaces and can be monitored through standard enterprise management tools.
```

Comment:
Maybe change to something like:

The Cuttlefish batching subsystem provides statistics on run-time characteristics of the components that make up the search system.